Sources/CorePayments/Networking/AnalyticsService.swift

```diff
@@ -40,6 +40,7 @@ public struct AnalyticsService {
     
     // MARK: - Internal Methods
     
+    /// Exposed to be able to execute this function synchronously in unit tests
```

Comment:
What's the support like for async await in XCTest?

Reply:
Support is solid! But in this case `sendEvent()` it isn't an `async` function since the feature clients don't care if it completed or not. It let us clean up the callsite code too. It doesn't return anything and just executes via a background Task. So in order for our unit tests to make assertions after `sendEvent` completes, I opted to expose the internal `async` function, which lets you do `await` on.

Though it's worth looking into to see if there's a way to manipulate `Tasks` (like [here](https://github.com/paypal/paypal-ios/blob/4e18740043971268993b600dea03a3e2e3dd3d01/Sources/CorePayments/Networking/AnalyticsService.swift#L36)) from within our test suite